src/main/scala/nl/knaw/dans/api/sword2/DepositHandler.scala

```diff
@@ -160,9 +150,9 @@ object DepositHandler {
     }
   }
 
-  def checkBagStoreBaseDir()(implicit id: String, bs: BagStoreBase): Try[Unit] = {
-    if (bs.isBagStoreAware) {
-      val baseDir = new File(bs.baseDir)
+  def checkBagStoreBaseDir()(implicit id: String, bs: Option[BagStoreSettings]): Try[Unit] = {
+    if (bs.nonEmpty) {
```

Comment:
I think that using nonEmpty is the most simple way and the most clear way to code this.  I looked into getOrElse and map and got constructions I was not happy with.  But perhaps I don't know how to do it neatly with map...